.prettierignore

```diff
@@ -0,0 +1,16 @@
+# Irrelevant files ignored for performance reasons.
+node_modules
+*.min.js
+**/lib/
+public/
+coverage/
+**/vendor/
+static
+_build
+# File types which Prettier supports but we don’t want auto-formatting.
+*.md
+# Files which contain incompatible syntax.
+*.html
+wagtail/admin/templates/wagtailadmin/edit_handlers/inline_panel.js
+wagtail/contrib/search_promotions/templates/wagtailsearchpromotions/includes/searchpromotions_formset.js
+wagtail/users/templates/wagtailusers/groups/includes/page_permissions_formset.js
```

Comment:
I cannot find this file in the codebase, I might be missing something but maybe this can be removed (from eslintignore also).

Reply:
Good spot – I had copied those straight from `.eslintignore`.